File: validstring.py
class Solution:
    def isValid(self, s: str) -> bool:
        if len(s) < 3:
            return False
        
        vowel=0
        consonent=0
        vowel_set="aeiouAEIOU"

        for c in s:
            if c.isalpha():
                if c in vowel_set:
                    vowel +=1
                else:
                    consonent +=1

            elif not c.isdigit():
                return False
    
        return vowel>=1 and consonent>=1

sol = Solution()
print(sol.isValid("234Adas"))

File: validstring.cpp
#include<iostream>
#include<string>
using namespace std;
class Solution{
public:
    bool isValid(string s){
        int n = s.length();
        if (n<3) return false;

        int vowel=0,consonent=0;
        string vowel_set="aeiouAEIOU";

        for (char c:s){
            if(isalpha(c)){
                if(vowel_set.find(c) != string::npos){
                    vowel++;
                }else{
                    consonent++;
                }
            }else if(!isdigit(c)){
                return false;
            }

        }
        return vowel>=1 && consonent>=1;

    }

};

int main(){
    Solution sol;
    cout << boolalpha;

    cout << sol.isValid("234Adas") << endl;
    return 0;
}